bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/dto/ThingStorageDTO.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.thing.dto;
+
+/**
+ * The {@link ThingStorageDTO} is a transfer object for Thing storage
```

Comment:
Isn't it a POJO used for mapping stored data? When using JPA such POJOs are called "Entities". It is also very common to keep entities internal so you can more easily change how data is stored without other code breaking.

Reply:
Good point, I will rename and move to an internal package.